lib/open_project/custom_styles/design.rb

```diff
@@ -46,9 +46,23 @@ def overridden?
     end
 
     ##
-    # Path the favicon
+    # Path to favicon
     def favicon_asset_path
-      'favicon.ico'.freeze
+      if Rails.env.development?
```

Comment:
Please add an opt-in flag here instead of just relying on the development mode. For developers that need to work on the favicons or related styles, it will introduce a new state that is impossible to disable.

Reply:
That is why it is a draft PR, do you mean the `OpenProject::Configuration` or something else?